packages/core/src/config/load.ts

```diff
@@ -106,54 +83,44 @@ export function findConfig(dir?: string): string | undefined {
   return existingConfigFiles[0];
 }
 
-export async function getConfig(
-  options: {
-    configPath?: string;
-    externalRefResolver?: BaseResolver;
-  } = {}
-): Promise<{
-  rawConfig: RawConfig;
+export async function getConfig(options: {
+  configPath?: string;
+  externalRefResolver?: BaseResolver;
+}): Promise<{
+  rawConfig?: RawUniversalConfig;
   document?: Document;
-  parsed?: Document['parsed'];
   resolvedRefMap?: ResolvedRefMap;
 }> {
-  const { configPath = findConfig(), externalRefResolver = new BaseResolver() } = options;
-  if (!configPath) return { rawConfig: {} };
+  if (!options.configPath) return {};
 
   try {
-    const { document, resolvedRefMap } = await resolveConfigFileAndRefs({
-      configPath,
-      externalRefResolver,
-    });
+    const { document, resolvedRefMap } = await resolveConfigFileAndRefs(options);
 
     const bundledRefMap = deepCloneMapWithJSON(resolvedRefMap);
-    const parsed = await bundleConfig(JSON.parse(JSON.stringify(document)), bundledRefMap);
+    const rawConfig = (await bundleConfig(
+      JSON.parse(JSON.stringify(document)),
+      bundledRefMap
+    )) as RawUniversalConfig;
 
     return {
-      rawConfig: transformConfig(parsed),
+      rawConfig,
       document,
-      parsed,
       resolvedRefMap,
     };
   } catch (e) {
-    throw new Error(`Error parsing config file at '${configPath}': ${e.message}`);
+    throw new Error(`Error parsing config file at '${options.configPath}': ${e.message}`);
   }
 }
 
 type CreateConfigOptions = {
-  extends?: string[];
   configPath?: string;
   externalRefResolver?: BaseResolver;
 };
 
 export async function createConfig(
-  config: string | RawUniversalConfig,
+  config?: string | RawUniversalConfig,
```

Comment:
If `config === undefined`, then the default `recommended` fallback will be applied.